src/commands/upgrade-2.0

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+readonly SCRIPT_PATH="/usr/local/bin/upgrade-2.0.0.sh"
+
+function print_help() {
+  echo "2.0 k8s migration script."
+  echo "This script outputs migrated values file on stdout (and saves it in new_values.yaml in current working directory)."
```

Comment:
> and saves it in new_values.yaml in current working directory

Is that true? I'd remove that.

Reply:
I mean - it's true for the script inside the container 🤔

Reply:
```suggestion
  echo "This script outputs migrated values file on stdout (and saves it in new_values.yaml in current working directory inside the container)."
```

Reply:
done